Functions/Import/xASL_imp_CheckImportSettings.m

```diff
@@ -64,15 +64,18 @@
 		nRightBrackets = nRightBrackets + numel(rightBracket);
 	end
 
-	nTokens = sum(x.modules.import.imPar.tokenOrdering>0);
+	nTokens = max(x.modules.import.imPar.tokenOrdering);
 
 	% Report this
 	fprintf('%s\n', [num2str(nLeftBrackets) ' tokens detected in folderHierarchy']);
 	fprintf('%s\n', [num2str(nTokens) ' tokens detected in tokenOrdering']);
 
-	if nLeftBrackets~=nTokens
-		fprintf('Note that the () symbols are reserved to define tokens in folderHierarchy!');
-		error('Number of tokens of folderHierarchy and tokenOrdering should be the same');
+	if nLeftBrackets < nTokens
+		error('The number of tokens in folderHierarchy defined by the () symbol should be equal to or higher than the length of the tokenOrdering vector.');
+	elseif nLeftBrackets > nTokens
+		warning('The number of tokens in folderHierarchy defined by the () symbol is higher than the length of the tokenOrdering vector. This is allowed, but make sure that this is what you want.');
+	elseif sum(x.modules.import.imPar.tokenOrdering>0) ~= nTokens
+		warning('Not all tokens from folderHierarchy appear to be used in the tokenOrdering vector. This is allowed, but make sure that this is what you want.')
```

Comment:
('Not all captured groups in folderHierarchy are used as a token in tokenOrdering. Ensure that this is correct')"